warnings.rb

```diff
@@ -1,9 +1,9 @@
 
-a = { "hello" => "world" }
+a={ "hello" => "world" }
 
 2 + 5
 
-
+"Oh this is better"
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Lint/Void: Literal `"Oh this is better"` used in void context.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.